requirements.txt

```diff
@@ -9,4 +9,4 @@ Werkzeug==1.0.1
 gunicorn==20.0.4
 bleach==3.2.1
 PyPDF2==1.26.0
-git+git://github.com/OpenSlides/openslides-auth-service@6e3a2a3b5d71334a273cb2fc39cc376e8e7290d6#egg=authlib&subdirectory=auth/libraries/pip-auth # authlib
+git+git://github.com/OpenSlides/openslides-auth-service@4f4dae3b0ed85e17086c5d5fccca8341c28d92c5#egg=authlib&subdirectory=auth/libraries/pip-auth # authlib
```

Comment:
This must be changed, if https://github.com/OpenSlides/openslides-auth-service/pull/55 is merged (the merge commit will have a different hash), so we'öl wait for the auth service first.

Reply:
Yeah, you're right, that the merge-commit has another hash, but the hash of the PR-commit is the same. 
Or do you want, that the commit-hash is the one from the merge-commit?

Reply:
> one from the merge-commit?

Definitly, since this commit belongs to the commits from `master`, the PR-commit is not (or not direct) coupled with master.

Reply:
Okay